### PR TITLE
feat: sections-landing

### PR DESCRIPTION
**Pull request by KambojRajan**

### CODE REVIEW

File: app/[locale]/institute/sections/page.tsx [hunk 1]
```diff
@@ -1,9 +1,91 @@
...
+    sports: MdOutlineSportsTennis,
+    store: MdOutlineStore,
+  };
+  const text = (await getTranslations(locale)).Section;
+
+  return (
+    <main className="container">
+      <Heading
+        glyphDirection={'dual'}
+        heading={'h1'}
```
Comment (GetPsyched) [inline app/[locale]/institute/sections/page.tsx:49]: Don't use the `{}` syntax when it's just a normal string
```suggestion
        glyphDirection="dual"
        heading="h1"
```

File: i18n/en.ts [hunk 2]
```diff
@@ -215,7 +215,7 @@ const text: Translations = {
   Section: {
     about: 'ABOUT',
     gallery: 'GALLERY',
-
+    title: 'Sections',
```
Comment (GetPsyched) [inline i18n/en.ts:218]: Make a new attribute `Sections` and put `title` in that. `Section` is reserved for the dedicated section page(s) we have.

File: app/[locale]/institute/sections/page.tsx [hunk 3]
```diff
@@ -1,9 +1,91 @@
...
+          className={cn(
+            'container mb-2 sm:mb-4 lg:mb-6 xl:mb-8',
+            'grid grid-cols-1 gap-2',
+            'sm:grid-cols-2 sm:gap-4',
+            'lg:grid-cols-3 lg:gap-6',
+            'xl:grid-cols-4 xl:gap-8'
+          )}
+        >
+          {sections.map((section, i) => {
+            const IconComponent = SectionIcons[section.urlName];
```
Comment (GetPsyched) [inline app/[locale]/institute/sections/page.tsx:63]: Kind of a small nit, but just `Icon` is fine
```suggestion
            const Icon = SectionIcons[section.urlName];
```

File: app/[locale]/institute/sections/page.tsx [hunk 4]
```diff
@@ -1,9 +1,91 @@
...
+
+export default async function Sections({
   params: { locale },
 }: {
   params: { locale: string };
 }) {
-  return <WorkInProgressStatus locale={locale} />;
+  const sections = await db.query.sections.findMany();
+
+  const SectionIcons: Record<string, IconType> = {
```
Comment (GetPsyched) [inline app/[locale]/institute/sections/page.tsx:30]: The convention for variable names is lower camel case
```suggestion
  const sectionIcons: Record<string, IconType> = {
```

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Padding is a little off for the "General Administration" section. Please check.

File: server/db/schema/club-event.schema.ts
Comment (GetPsyched) [inline server/db/schema/club-event.schema.ts:1]: Remove this

File: server/db/schema/event.shema.ts
Comment (GetPsyched) [inline server/db/schema/event.shema.ts:1]: And remove this

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/[locale]/institute/sections/page.tsx [hunk 5]
```diff
@@ -1,9 +1,93 @@
...
   params: { locale },
 }: {
   params: { locale: string };
 }) {
-  return <WorkInProgressStatus locale={locale} />;
+  const sections = await db.query.sections.findMany();
+  console.log('section =>', sections);
+
+  const sectionIcons: Record<string, IconType> = {
+    accounts: MdOutlineAccountBalance,
```
[the remark this answers is not in the document]
Comment (KambojRajan) [reply app/[locale]/institute/sections/page.tsx:33]: sir this is being done by Prettier

**Pull request closed without merging**